frontend/packages/console-app/src/components/detect-namespace/namespace.ts

```diff
@@ -64,28 +47,31 @@ export const useValuesForNamespaceContext = () => {
     }
     // Only run this hook after all resources have loaded.
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [resourcesLoaded, useProjects]);
+  }, [resourcesLoaded]);
```

Comment:
won't need `useProjects` as well in the dependency list of effect?

Reply:
As long as useProjects is pending the resourcesLoaded is false. It switches to true if everything is loaded (useProjects and 2 user settings). So the useEffect must not be called earlier.

The condition in the first line would ignore this extra call, but I think useProjects is not required here.

Reply:
got it 👍